test-main.js

```diff
@@ -0,0 +1,65 @@
+var allTestFiles = ['assert', 'underscore'];
+var TEST_REGEXP = /tests\/.*\.js$/i;
+
+// Automatically clenaup sinon spies and stubs.
+var realSinon = sinon;
+beforeEach(function() {
+    sinon = realSinon.sandbox.create();
+});
+afterEach(function() {
+    sinon.restore();
+    sinon = realSinon;
+});
+
+function withSettings(changes, test) {
```

Comment:
So I tried using Squire instead of this and I wasn't crazy about it. I updated the `tests/models.js` file. We could probably work out another helper for settings that would make it a bit shorter but I found it a bit awkward.

My changes are here: https://github.com/mozilla/marketplace-core-modules/blob/squire/tests/models.js.

This essentially causes each test to reload the dependencies which caused the file to run in ~0.8s locally. Using `withSettings` as is the tests run in ~0.04s. So it made that file 20 times slower.